views/createAccount.pug

```diff
@@ -7,29 +7,15 @@ include include/sauce-common
     | SCCS accounts are available to all Swarthmore students, faculty,
     | and staff&mdash;and you'll still be able to access it after
     | graduation.
-  form#createForm.form-signin.needs-validation(action='/account/create', method='post', novalidate)
+  form#createForm.form-signin.needs-validation(action='/account/init', method='post', novalidate)
+    input(type='hidden', name='id', value=id)
+    input(type='hidden', name='key', value=key)
```

Comment:
The createAccount page itself (the one that doesn't contain Cygnet data) doesn't need `id` and `key` hidden form parameters. Those will go in the other page that requires you to actually click a reset link.